### PR TITLE
Updated es and es-419 translations

### PR DESCRIPTION
**Pull request by Gudahtt**
Explanation: Updated es and es-419 translations.

Manual testing steps: Just change to Spanish or Spanish (Latin America) language configuration setting.

(This PR was originally created by @fernandospr in #10073, which was accidentally closed)

### CODE REVIEW

**Comment by Gudahtt** [comment]
If any community contributors would like to help review this PR, it would be greatly appreciated!

I'll tag anyone here that has contributed in the past: @Pedro-vk @mapachurro @jonathansmirnoff @raulk @vimalsagar @hermanjunge @omartinex 

I wanted to give you all the opportunity to provide feedback because we owe our current translations to you. Please don't feel obliged if you don't have the time to review, and please let me know if you'd like me to stop tagging you for reviews in the future.

Thanks!

**Comment by Gudahtt** [comment]
This PR has been rebased onto `develop` to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

File: app/_locales/es/messages.json [hunk 1]
```diff
@@ -26,9 +34,18 @@
...
+    "message": "Activo"
+  },
+  "activity": {
+    "message": "Actividad"
+  },
   "activityLog": {
     "message": "registro de actividades"
   },
+  "addAccount": {
+    "message": "Agregar una cuenta"
```
Comment (ricardolyn) [inline app/_locales/es/messages.json:47]: Maybe I would use more "Añadir una cuenta". need confirmation by 1 more person to see if is worth changing

File: app/_locales/es/messages.json [hunk 2]
```diff
@@ -163,32 +281,94 @@
...
+  "connectToMultiple": {
+    "message": "Conectar a $1",
+    "description": "$1 will be replaced by the translation of connectToMultipleNumberOfAccounts"
+  },
+  "connectToMultipleNumberOfAccounts": {
+    "message": "$1 cuentas",
+    "description": "$1 is the number of accounts to which the web3 site/application is asking to connect; this will substitute $1 in connectToMultiple"
+  },
+  "connectWithMetaMask": {
+    "message": "Conectar Con MetaMask"
```
Comment (ricardolyn) [inline app/_locales/es/messages.json:314]: "Conectar con MetaMask"

File: app/_locales/es/messages.json [hunk 3]
```diff
@@ -238,24 +424,56 @@
...
     "message": "Decimales de precisión"
   },
   "decimalsMustZerotoTen": {
     "message": "Los decimales deben ser al menos 0 y no más de 36"
   },
+  "decrypt": {
+    "message": "Descifrar"
+  },
+  "decryptCopy": {
+    "message": "Copiar mensaje cifrado"
```
Comment (ricardolyn) [inline app/_locales/es/messages.json:446]: Shouldn't be instead "Copiar mensaje decifrado"?

File: app/_locales/es/messages.json (hunk 2 above)
Comment (fernandospr) [reply app/_locales/es/messages.json:314]: Yes, however, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L314) is "Connect **W**ith Metamask".

File: app/_locales/es/messages.json (hunk 3 above)
Comment (fernandospr) [reply app/_locales/es/messages.json:446]: I'm not sure about this, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L449) is "Copy encrypted message". So I guess "Copiar mensaje cifrado" or "Copiar mensaje encriptado" should be ok.

Comment (ricardolyn) [reply app/_locales/es/messages.json:446]: got confused by the id then. in that case, is ok.

File: app/_locales/es/messages.json (hunk 2 above)
Comment (ricardolyn) [reply app/_locales/es/messages.json:314]: seems strange but if that is that is how we are using with the brand, then ok.

Comment (kumavis) [reply app/_locales/es/messages.json:314]: "Connect With Metamask" means the dapp can read your ethereum address

File: app/_locales/es/messages.json (hunk 1 above)
Comment (Gudahtt) [reply app/_locales/es/messages.json:47]: I think "Agregar" makes sense here? At least if this comment is accurate: https://www.reddit.com/r/duolingo/comments/1alunr/what_is_the_difference_between_a%C3%B1adir_and_agregar/c8ylfwv/
The "Add" in this case is more the "add to a group/aggregate" case I think. Each account is independent.

**Comment by Gudahtt** [comment]
I've rebased this PR and made a couple of minor lint fixes.

@ricardolyn Thanks so much for the review! Just wondering - did you finish reviewing the `es` locale? Did you review `es-419` as well?

**Comment by ricardolyn** [comment]
yes, I finished. np 👍

**Pull request merged**